### ISSUE TITLE
Bulk purchase pricing

### ISSUE THREAD

**Issue by emergingdzns** [Attendize/Attendize#431]
I have a client that is using a registration system I put together years ago built on CodeIgniter. It works ok but I'd love to switch them over to Attendize. Attendize has the majority of what I need. In addition to the current setup I need discount/coupon codes (already in another issue, though that issue is closed), sending tickets to attendees (also in another issue) and finally I need the ability to do bulk ticket pricing. This client has events that are attended online all over the world had often there are thousands of tickets sold. Many times administrators of organizations will purchase numerous tickets all at once and this client likes to offer tiered pricing for buying in bulk. Something like the following:

1 - 5 tickets = regular price
5 - 10 tickets = 10% discount automatically applied
10 - 20 tickets = 20% discount automatically applied
20+ tickets = 50% discount automatically applied

The discounts applied don't require a code. However, any coupon codes should still be an option during checkout if that feature is available or any codes are available for that event.

I'm not sure what it would take to build something like this in. Just a suggestion from a likely buyer. Thanks!

**Comment by jeremyquinton** [comment]
@emergingdzns 

With regards to discount codes this is something that is on the road map e.g discount a certain ticket by a certain amount. However I can't say exactly when this feature will be complete. With regards to doing bulk ticket pricing we could probably build this in at the same time though I can't guarantee it. 
To build discount codes in correctly we will need to change the domain model slightly. Once that is done any type of discounts should be fairly trivial.

With regards to sending tickets currently Attendize only sends the tickets to the purchaser. In the current domain model we store each attendee separately.  Should be fairly simple to add a feature that when ordering have a tickbox that prompts the user if they would like to send the tickets individually and then send them individually when order is completed.
